content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -133,6 +142,42 @@ This requires you have `openssl` installed on your machine.
     /tmp/pod-cert-chain-workload.pem: OK
     {{< /text >}}
 
+## (Optional) Generating root and intermediate keys and certificates offline
+
+This section gives you instructions on how to generate the keys and certificates offline and apply to
+the Istio cluster.
+
+On an offline machine with good security (restricted access), follow the `Create the root pair` and the
```

Comment:
`1. On an machine with good security and rerstricted access, use [this guide](link) to create a root par and intermediate pair.1